Assets/IPieceState.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class IPieceState : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
```

Comment:
void Start() and void Update() are methods of MonoBehavior. Since we aren't planning on extending from monobehavior we can remove them. What we do want to have here is a method header for our validatePiece() method that our piece states are extending. Go ahead and add that method header to this file.